recipes/ecell4_base/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+${PYTHON} setup.py install;
```

Comment:
Can you use `${PYTHON} -m pip install -vv .` after adding `pip` to requirements?

Reply:
I replaced `setup.py install` to `-m pip install -vv .`